### PR TITLE
KIALI-2294 add docs to tell the user they will be prompted for credentials and g…

### PR DESCRIPTION
**Pull request by jmazzitelli**
This also gives a link to the script so they can see what other env vars they can use.
Also - don't point to preliminary istio docs - the istio main docs page has the kiali docs now

### CODE REVIEW

File: content/gettingstarted/index.adoc [hunk 1]
```diff
@@ -48,6 +48,8 @@ icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in yo
 curl -L http://git.io/getLatestKiali | bash
 ----
 
+Be default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKiali[the comments at the top of the install script].
```
[the remark this answers is not in the document]
Comment (jmazzitelli) [reply content/gettingstarted/index.adoc]: opps :) yes, you are correct. I committed a fix for this typo.

**Comment by abonas** [comment]
can you squash the commits?

**Comment by jmazzitelli** [comment]
> can you squash the commits?

Done. But I thought during review process, we keep our individual commits so reviewers can easily check what was changed from the last review. When we are ready to merge, the merger can click the "Squash and merge" button in github and get it down to 1 squashed commit.

**Comment by abonas** [comment]
> Done. But I thought during review process, we keep our individual commits so reviewers can easily check what was changed from the last review. When we are ready to merge, the merger can click the "Squash and merge" button in github and get it down to 1 squashed commit.

I was not aware of this, good to know. I am actually used to work differently from other projects.

File: content/gettingstarted/index.adoc [hunk 2]
```diff
@@ -48,6 +48,8 @@ icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in yo
 curl -L http://git.io/getLatestKiali | bash
 ----
 
+By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKiali[the comments at the top of the install script].
```
Comment (JStickler) [inline content/gettingstarted/index.adoc]: "By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali."

This is a bit wordy, we can probably trim it down to:

"By default, the installation script will prompt you to enter a username and passphrase  that you want users to enter to log in to Kiali."
